cytotable/utils.py

```diff
@@ -107,13 +108,13 @@ def _parsl_loaded() -> bool:
     try:
         # try to reference Parsl dataflowkernel
         parsl.dfk()
-    except RuntimeError as rte:
-        # if we detect a runtime error that states we need to load config
+    except ConfigurationError as pce:
+        # if we detect a Parsl ConfigurationError that states we need to load config
         # return false to indicate parsl config has not yet been loaded.
-        if str(rte) == "Must first load config":
+        if str(pce) == "Must first load config":
```

Comment:
This seems a bit brittle to me. Maybe ok for now, but does `ConfigurationError` have a text attribute to check?

Reply:
Thank you for this, I'll move to use `pce.args[0]` which represents the string message sent to the Exception. I explored the use of [`traceback`](https://docs.python.org/3/library/traceback.html) for more isolated, declarative treatment here but I couldn't justify a switch there as less complex.